src/test/java/uk/gov/hmcts/darts/common/datamanagement/api/DataManagementFacadeImplTest.java

```diff
@@ -74,7 +80,11 @@ class DataManagementFacadeImplTest {
     @Mock
     private UserAccountRepository userAccountRepository;
     @Mock
+    private ArmApiServiceImpl armApiService;
+    @Mock
     private BlobClient blobClient;
+    @Mock
+    private DownloadResponseMetaData downloadResponseMetaDataMock;
```

Comment:
comment on naming convention - none of the other mocks use "Mock" in the name - maybe consider following same pattern?

Reply:
apologies John no idea how I didn't see your comment and I merged. 
I'm going to take note and make the change on my next PR. Thanks

Reply:
hi John I tried but `downloadResponseMetaData` is already defined in the same scope